prototypes/entity/power/heat-source.lua

```diff
@@ -18,25 +18,25 @@ local inputs = {
 }
 
 local tier_map = {
-    ["burner-reactor"] = { tier = 1, prog_tier = 3, icon_name = "heat-source-burner", material = "base" },
-    ["burner-reactor-2"] = { tier = 2, prog_tier = 4, icon_name = "heat-source-burner", material = "silver-aluminum" },
-    ["burner-reactor-3"] = { tier = 3, prog_tier = 5, icon_name = "heat-source-burner", material = "gold-copper" },
-    ["fluid-reactor"] = { tier = 1, prog_tier = 3, icon_name = "heat-source-fluid", material = "base" },
-    ["fluid-reactor-2"] = { tier = 2, prog_tier = 4, icon_name = "heat-source-fluid", material = "silver-aluminum" },
-    ["fluid-reactor-3"] = { tier = 3, prog_tier = 5, icon_name = "heat-source-fluid", material = "gold-copper" },
+    ["bob-burner-reactor"] = { tier = 1, prog_tier = 3, icon_name = "heat-source-burner", material = "base" },
+    ["bob-burner-reactor-2"] = { tier = 2, prog_tier = 4, icon_name = "heat-source-burner", material = "silver-aluminum" },
+    ["bob-burner-reactor-3"] = { tier = 3, prog_tier = 5, icon_name = "heat-source-burner", material = "gold-copper" },
+    ["bob-fluid-reactor"] = { tier = 1, prog_tier = 3, icon_name = "heat-source-fluid", material = "base" },
+    ["bob-fluid-reactor-2"] = { tier = 2, prog_tier = 4, icon_name = "heat-source-fluid", material = "silver-aluminum" },
+    ["bob-fluid-reactor-3"] = { tier = 3, prog_tier = 5, icon_name = "heat-source-fluid", material = "gold-copper" },
 }
 
 if reskins.lib.version.is_same_or_newer(mods["bobpower"], "1.1.6") then
```

Comment:
Need to create an issue to find and excise all the various version checks like this. We'll use 2.0 as a new baseline and drop compatibility with everything older.